python/cac_tripplanner/templates/base.html

```diff
@@ -23,7 +23,7 @@
     <![endif]-->
   </head>
 
-  <body class="homepage">
+  <body>
```

Comment:
If we remove the `homepage` class here, I don't think the contents of `_homepage.scss` will apply to anything.